docs/integrations/sources/amazon-seller-partner-migrations.md

```diff
@@ -0,0 +1,21 @@
+# Amazon Seller Partner Migration Guide
+
+## Upgrading to 2.0.0
+
+This change removes Brand Analytics from Airbyte Cloud version of connector and permanently removes deprecated FBA reports.
```

Comment:
This change removes Brand Analytics and permanently removes deprecated FBA reports (from Airbyte Cloud).

Reply:
To be more precise:
Brand Analytics removed from CLOUD version only (will still be available for OSS users).
FBA reports removed for both CLOUD and OSS.

Which comment should be left?

Reply:
done